tools/tertiary-analysis/scanpy/scanpy-neighbours.xml

```diff
@@ -13,14 +13,14 @@ PYTHONIOENCODING=utf-8 scanpy-neighbours.py
     -o output.h5
     -F '${output_format}'
     #if $settings.default == "false"
-        -N '${settings.n_neighbour}'
+        -N '${settings.n_neighbours}'
```

Comment:
Should we use American's spelling consistently (scanpy does so)? I see `flavour` was changed to `flavor` in find-clusters, so perhaps `n_neighbours` -> `n_neighbors`?

Reply:
What is there on scanpy-scripts? nf uses `n_neighbours` but if scanpy-scripts uses `n_neighbors` I would go for it. scanpy-scripts is what we don't want to change, galaxy or nextflow can be changed.